build-system/tasks/issues-tasks.js

```diff
@@ -0,0 +1,255 @@
+var BBPromise = require('bluebird');
+var argv = require('minimist')(process.argv.slice(2));
+var assert = require('assert');
+var child_process = require('child_process');
+var config = require('../config');
+var extend = require('util')._extend;
+var fs = require('fs-extra');
+var git = require('gulp-git');
+var gulp = require('gulp-help')(require('gulp'));
+var request = BBPromise.promisify(require('request'));
+var util = require('gulp-util');
+
+var GITHUB_ACCESS_TOKEN = process.env.GITHUB_ACCESS_TOKEN;
+var exec = BBPromise.promisify(child_process.exec);
+var gitExec = BBPromise.promisify(git.exec);
+
+var isDryrun = argv.dryrun;
+var verbose = (argv.verbose || argv.v);
+
+const issuesopt = {
+  url: 'https://api.github.com/repos/ampproject/amphtml/issues',
+  headers: {
+    'User-Agent': 'amp-changelog-gulp-task',
+    'Accept': 'application/vnd.github.v3+json'
+  },
+};
+
+const optionsMilestone = {
+  url: 'https://api.github.com/repos/ampproject/amphtml/milestones',
+  headers: {
+    'User-Agent': 'amp-changelog-gulp-task',
+    'Accept': 'application/vnd.github.v3+json'
+  },
+};
+
+if (GITHUB_ACCESS_TOKEN) {
+  issuesopt.qs = {
+    access_token: GITHUB_ACCESS_TOKEN
+  }
+}
+
+if (GITHUB_ACCESS_TOKEN) {
+  optionsMilestone.qs = {
+    access_token: GITHUB_ACCESS_TOKEN
+  }
+}
+
+function processissues() {
+  if (!GITHUB_ACCESS_TOKEN) {
+    util.log(util.colors.red('You have not set the ' +
+        'GITHUB_ACCESS_TOKEN env var.'));
+    util.log(util.colors.green('See https://help.github.com/articles/' +
+        'creating-an-access-token-for-command-line-use/ ' +
+        'for instructions on how to create a github access token. We only ' +
+        'need `public_repo` scope.'));
+    return;
+  }
+
+  return getGitMetadata();
+}
+
+function getGitMetadata() {
+return getIssues()
+.then(function() {
+	util.log(util.colors.blue('automation applied'));
+	if (isDryrun) {
```

Comment:
What does this statement on `dryrun` do here?

Reply:
took it out